src/controllers/hashPassword.js

```diff
@@ -0,0 +1,13 @@
+const bcrypt = require('bcrypt');
```

Comment:
This file should not exist in the controller directory.
It should be part of a separate concern called ```middlewares``` or maybe ```helpers```.